CONTRIBUTING.md

```diff
@@ -14,12 +14,16 @@ Thanks for your interest in contributing to Turbo!
 Dependencies
 
 1.  On OSX: `brew install sponge jq protobuf protoc-gen-go protoc-gen-go-grpc golang`
-1.  Run `pnpm install` at root
+2.  Setup Rust via [rustup](https://rustup.rs/): `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
```

Comment:
We could add a step saying to choose nightly instead of stable

Reply:
Since we have a `rust-toolchain` file in the repo, cargo will automatically install and use the nightly version when users run it here. I think it's reasonable for them to use the default install for this.